### ISSUE TITLE
[VT-2] vit; arrow-key weirdness while adding a task

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#24]
_David Patrick on 2013-09-07T03:49:25Z says:_ 

 I know there's little reason to do it, but if one hits "a" and Adds a task, then while editing hits the arrow-keys, weird things happen;
*  up-arrow clears the entry and replaces it with "Annotate:"
*  right-arrow clears the entry
*  left-arrow clears the entry and replaces it with "Delete task? y/n"

what should happen? well.. not that! :) up/down begs for command-history, and left/right should move the cursor position.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-09-07T03:49:25Z
Modified: 2014-02-09T01:14:57Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-10-04T19:19:08Z says:_ 

 I've been working on this. Several parts of VIT are affected by this issue. The problem is that currently VIT is viewing each character as a separate key. This is not right because the arrow key keycode is composed of several keycodes (one of which is an escape and one of which is an 'a' which is why you see "annotation"). I have a simple idea for fixing this that I've been testing the last couple of weeks. It will at least be an improvement and then we can refine it if needed.

I will close this issue once I fix the arrow key weirdness. Regarding the command history and cursor movement, please open up separate feature requests.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-10-07T03:09:58Z says:_ 

 Fixed at 916141d5.
